### DISCUSSION TITLE
skf接口导出公钥转pem问题

### DISCUSSION THREAD

**Discussion by b1gcat** [emmansun/gmsm#32] [Q&A]
typedef struct Struct_ECCPUBLICKEYBLOB{
	ULONG	BitLen;
	BYTE	XCoordinate[ECC_MAX_XCOORDINATE_BITS_LEN/8];
	BYTE	YCoordinate[ECC_MAX_YCOORDINATE_BITS_LEN/8];
}ECCPUBLICKEYBLOB, *PECCPUBLICKEYBLOB;

想吧这个转为pem格式的pubkey，有参考的说明吗

**Comment by emmansun** [comment]
这个你可以参考 http://gmssl.org/docs/skf.html
golang的话，你可以参考 https://github.com/emmansun/gmsm/blob/8bf00b0a84f0b78d3684e1c76ef0cc8b5a3b7753/smx509/x509.go#L107
